docs/UserGuide.md

```diff
@@ -13,14 +13,41 @@ ContactMate is a **desktop app for managing clients at Active Ageing Centres (AA
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
+   1. Open a command terminal.
+      1. For Windows Users:
+         1. Hold down the `Windows` key and press `R`(<kbd>&#x229E; Win</kbd> + <kbd>R</kbd>).
+         2. Type `cmd` and press `Enter`.
+      2. For Mac Users:
+         1. Hold down the `Command` key and press `Space` (<kbd>&#x2318; Cmd</kbd> + <kbd>Space</kbd>).
+         2. Type `Terminal` and press `Enter`.
+   1. In the terminal, type `java --version` and press `Enter`.
+   2. If you have Java `17` or above installed, you should see a message like this. Referencing the first line (`openjdk XX.0.12 2024-07-16 LTS`), `XX` should be `17` or above.
+    ```
+       openjdk 17.0.12 2024-07-16 LTS
+       OpenJDK Runtime Environment Corretto-17.0.12.7.1 (build 17.0.12+7-LTS)
+       OpenJDK 64-Bit Server VM Corretto-17.0.12.7.1 (build 17.0.12+7-LTS, mixed mode, sharing)
+      ```
+    3. If you do not have Java `17` or above installed, proceed to Step 2, otherwise, skip to Step 3.
+2. Install Java `17` using the following steps.
    1. Go to [this link](https://www.oracle.com/java/technologies/javase-jdk17-downloads.html).
    2. Download the appropriate installer for your operating system.
    3. Run the installer and follow the instructions to install Java.
 1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14b-3/tp/releases).
-
+   1. You can find the `.jar` file under the Assets section of the latest release.
+   1. Click on the file `contactmate.jar` to download it.
+   ![JarFileImage.png](images%2FJarFileImage.png)
 1. Copy the file to the folder you want to use as the _home folder_ for ContactMate.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar contactmate.jar` command to run the application.<br>
+1. Open a command terminal for the folder you put the jar file in by following these steps (Folder is named `folder with contactmate jar` in the examples below) :
+   1. For Windows Users:
+      1. Right-click on the folder.
+      1. Click on `Open in Terminal` as highlighted below.
+        ![WindowsCmdOpen.png](images%2FWindowsCmdOpen.png)
+   1. For Mac Users:
+      1. Hold down the `Control` key and click on the folder (<kbd>Control</kbd> + <kbd>Click</kbd>).
+      1. Click on `New Terminal at Folder` as highlighted below.
+         ![MacCmdOpen.jpg](images%2FMacCmdOpen.jpg)
+1. In the command terminal you just opened in Step 5, type the `java -jar contactmate.jar` command and press Enter to run the application.<br>
```

Comment:
maybe can write type `java -jar contact mate.jar` and press Enter instead so its shorter and less confusing?